pkg/datamover/datamover.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/pkg/errors"
 	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
 	"golang.org/x/sync/errgroup"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
NIT: rename the import alias as `apierrors` , keeps things consistent with the code base